fdbrpc/BlobStore.actor.cpp

```diff
@@ -953,7 +953,14 @@ ACTOR Future<std::string> uploadPart_impl(Reference<BlobStoreEndpoint> bstore, s
 		throw checksum_failed();
 
 	// No etag -> bad response.
-	std::string etag = r->headers["ETag"];
+	std::string etag;
+
+	for (const auto &h : r->headers) {
+		if(0 == strcasecmp(h.first.c_str(), "etag")) {
```

Comment:
strcasecmp is stricmp on windows, so I think this is going to fail to compile there.  You can probably use [`boost::iequals`](https://www.boost.org/doc/libs/1_68_0/doc/html/boost/algorithm/iequals.html) instead?

Reply:
No problem, replaced to boost::iequals